modules/yarpc/yarpc.go

```diff
@@ -281,16 +247,17 @@ func newYARPCModule(
 		statsClient: newStatsClient(host.Metrics()),
 		log:         ulog.Logger(context.Background()).With(zap.String("module", host.Name())),
 	}
-	if err := host.Config().Scope("modules").Get(host.Name()).PopulateStruct(&module.config); err != nil {
-		return nil, errs.Wrap(err, "can't read inbounds")
-	}
+
+	val := host.Config().Scope("modules").Get(host.Name()).Value()
```

Comment:
Should we get service name rather than using `host.Name()` here? Also, scoped config doesn't add any value here.